### PR TITLE
Finex support

### PR DESCRIPTION
**Pull request by calj**

### CODE REVIEW

File: templates/config/barong/seeds.yml.erb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    description: "User entered a valid code from sms"
+  - key: document
+    value: verified
+    description: "User personal documents have been verified"
+
+permissions:
+# SUPER ADMIN has an access to the whole system without any limits
+# so we can just grant him access to /api/v2/*
+ - { role: superadmin, verb: all, path: api/v2, action: accept }
+ - { role: superadmin, verb: post, path: api/v2/peatio/market/orders, action: drop }
```
Comment (calj) [inline templates/config/barong/seeds.yml.erb]: finex condition is missing here

File: templates/config/barong/seeds.yml.erb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+ - { role: support, verb: all, path: api/v2/barong/admin/permissions, action: drop }
+ # we are watching after you 0_0
+ - { role: support, verb: all, path: admin, action: audit }
+ - { role: support, verb: all, path: api/v2, action: audit }
+
+# TRADER
+ # has standard user permissions to ACCOUNT, MARKET, RESOURCE modules
+ - { role: trader, verb: all, path: api/v2/peatio/account, action: accept }
+<%- if @config['finex']['enabled'] -%>
+ - { role: trader, verb: get, path: api/v2/peatio/market, action: accept }
```
Comment (calj) [inline templates/config/barong/seeds.yml.erb]: we need to explicitly drop the post here

File: templates/config/barong/seeds.yml.erb (hunk 1 above)
Comment (dnfd) [reply templates/config/barong/seeds.yml.erb]: It was before, I think superadmin should not trade

Comment (calj) [reply templates/config/barong/seeds.yml.erb]: I would keep allowing him by default, this is useful for testing, then each platform can still block it.

**Pull request merged**